Cargo.toml

```diff
@@ -10,13 +10,13 @@ crate-type = ["rlib", "cdylib"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-simple-logging = "2.0.2"
-log = "0.4.17"
-winreg = "0.10.1"
-jxl-oxide = "0.5.1"
+simple-logging = "2.0"
+log = "0.4"
```

Comment:
Was this done by some tool or by hand? Not sure the point to remove the patch versions.

Reply:
This was done by hand. Since the logging crates are stable I removed the patch versions. Maybe jxl-oxide should have it just to be sure. As you wish!